.github/dependabot.yml

```
@@ -0,0 +1,37 @@
+# Dependabot configuration file
+# Documentation: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+#
+# DEMO FILE: Copy to .github/dependabot.yml in your target repository.
+# This file demonstrates real-world Dependabot configuration patterns
+# used in production environments.
+
+version: 2
```

actionlint (1.7.12)

[error] "jobs" section is missing in workflow

(syntax-check)